welcome/run-d-program-locally.md

```diff
@@ -0,0 +1,49 @@
+# D Programme lokal ausführen
+
+Eine D Installation beinhaltet unter anderem den D Compiler `dmd`,
+ein Skripttool `rdmd` und den Packetmanager `dub`.
+
+### DMD Compiler
+
+Der *DMD* Compiler kompiliert D Dateien und erstellt eine Binärdatei.
+Auf der Kommandozeile kann *DMD* mit dem Dateinamen aufgerufen werden:
+
+    dmd hallo.d
+
+Es gibt viele Optionen, die das Verhalten des *DMD* Compilers verändern.
+Diese sind in der [Online-Dokumentation](https://dlang.org/dmd.html#switches))
+oder per `dmd --help` zu finden.
+
+### Fliegende Kompilierung mit `rdmd`
+
+Das Programm `rdmd`, welches zusammen mit dem DMD Compiler angeboten wird,
+sorgt dafür, dass alle Abhängigkeiten kompiliert sind und führt die kompilierte
+Datei automatisch aus:
+
+    rdmd hallo.d
+
+Auf UNIX System kann das Shebangkommando `#!/usr/bin/env rdmd` als erste Zeile
+in einer D-Quelldatei mit Ausführberechtigungen eingefügt werden und erlaubt eine
+Ausführung der Datei analog zu Skriptsprachen.
+
+Weitere Informationen sind in der [Online-Dokumentation](https://dlang.org/rdmd.html)
+oder per `rdmd --help` zu finden.
+
+### Paketmanager `dub`
```

Comment:
> I would go with Packetmanager on all of them

Good catch, while it's not part of the Duden I would go with "Paketmanager" as the Wikipedia page is also about "Paketverwaltung":

https://de.wikipedia.org/wiki/Paketverwaltung